submissions/tosinKolawole/HTML/index.html

```diff
@@ -0,0 +1,53 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset=@"UTF-8">
+    <title>Infinix Note series</title>
+</head>
+<body>
+      <header>
+            <h1><b>INFINIX NOTE 5</b></h1>
+            <h4>The most amazing phone yet in the Infinix note series.</h4>
+       </header>
+      <figure>
+        <ul> 
+        <img src="https://static.toiimg.com/photo/65514147/Infinix-Note-5.jpg">
+        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQdelXiyfO3_dwgqs7wdnbXrWLOH8VfH7A70A&usqp=CAU">
+        </ul>
+      </figure>
+     <!--This section is for product price-->
+      <section>
+          <h2><b>Product Price<b></h2>
+          <article>
+             <p> 
+              The Infinix Note 5 Price in Nigeria is best on Beta Market Online stores and its current price starts at N57,000 naira for the 3GB variety and N65,000 naira for the 4GB variety.    
```

Comment:
This line of code is too long. Kindly use the enter key to break it to the next line.